### ISSUE TITLE
it crashes when editing channels

### ISSUE THREAD

**Issue by Miguelwan** [Fleker/CumulusTV#192]
I love Cumulus, nice job Fleker.
After the update of two days ago, the app crashes every time I want to edit a channel. The channels added after that update crash the app when I try to open them (does not even gives the option of play or edit).
After the update of today, the app crashes when I try to add a channel.
I use nvidia shield and add all the channels manually (I have never been able to make it work with a Json in googledrive).

**Comment by Fleker** [comment]
Yep:

    Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference
       at com.felkertech.n.plugins.MainPicker$TvPickerDialog.show(MainPicker.java:657)

**Comment by Fleker** [comment]
Seems like it's missing in the tablet layout, which is a blend of phone and TV. Not sure why that is causing a problem though.

**Comment by Fleker** [comment]
The `-television` layout qualifier doesn't seem like it has high enough priority compared to `-w600`

**Comment by Fleker** [comment]
Fixed in `1.6.7`, test developed to make sure it is stable

**Comment by farhanito** [comment]
The issue still persist in my Sony Bravia

**Comment by josuearisty** [comment]
Same here! And for some unknown reason when Im watching channels via cumulus, it will have random crashes, causing even Live Channels to force close.

**Comment by Miguelwan** [comment]
In my case the editing and creating crash was solved, but I realized something the other day. When a channel is created using the app (not modifying the json in google drive), if some info is left empty, then the app put no data in the place of the info in the metadata, for example: "logo":""
This makes the app to crash when I try to play or edit the new channel. I erased that data in the json file in all the new channels (in the previous example it would be "logo":"" what I would erase). After this I redownload the channel list and the new channels were working properly.
